src/puyapy-stubs/op.pyi

```diff
@@ -2200,1458 +2284,826 @@ class Global:
         Stack: [...] -> [..., X]
         TEAL: global F
 
-        :returns UInt64: Number of transactions in this atomic transaction group. At least 1
-        """
-    @staticmethod
-    def logic_sig_version() -> UInt64:
+        :returns UInt64: The remaining cost that can be spent by opcodes in this program.
         """
-        global field F
-
-        Groups: Loading Values
+    caller_application_id: UInt64 = ...
+    """
+    global field F
+    
+    Groups: Loading Values
+    
+    Stack: [...] -> [..., X]
+    TEAL: global F
+    
+    """
 
-        Stack: [...] -> [..., X]
-        TEAL: global F
+    caller_application_address: Account = ...
+    """
+    global field F
+    
+    Groups: Loading Values
+    
+    Stack: [...] -> [..., X]
+    TEAL: global F
+    
+    """
 
-        :returns UInt64: Maximum supported version
-        """
-    @staticmethod
-    def round() -> UInt64:
-        """
-        global field F
+    asset_create_min_balance: UInt64 = ...
+    """
+    global field F
+    
+    Groups: Loading Values
+    
+    Stack: [...] -> [..., X]
+    TEAL: global F
+    
+    """
 
-        Groups: Loading Values
+    asset_opt_in_min_balance: UInt64 = ...
+    """
+    global field F
+    
+    Groups: Loading Values
+    
+    Stack: [...] -> [..., X]
+    TEAL: global F
+    
+    """
 
-        Stack: [...] -> [..., X]
-        TEAL: global F
+    genesis_hash: Bytes = ...
+    """
+    global field F
+    
+    Groups: Loading Values
+    
+    Stack: [...] -> [..., X]
+    TEAL: global F
+    
+    """
 
-        :returns UInt64: Current round number. Application mode only.
-        """
-    @staticmethod
-    def latest_timestamp() -> UInt64:
-        """
-        global field F
+class InnerTransaction:
+    """Functions for the ops: `itxn`, `itxnas`"""
 
-        Groups: Loading Values
+    sender: Account = ...
```

Comment:
These maybe shouldn't be properties? They're not constants.

Reply:
Good point, done